draft-deeglaze-amd-sev-snp-corim-profile.md

```diff
@@ -202,10 +202,10 @@ The `R[lo:hi]` notation will reference the attestation report byte slice from of
 The `leuintN` type is another name for a byte string, but with the interpretation that it represents an unsigned integer with `N` bit width.
 
 **mkey 0**: VERSION.
-Expressed as `&(raw-value: 4): tagged-leuint32`.
+Expressed as `&(version: 0): int-version-map`.
```

Comment:
Hi @deeglaze, the CoRIM spec says (and like you mentioned in the meeting also) that the version codepoint details the versioning of a measured environment. Is it also suitable for mkey 0, which defines the ATTESTATION_REPORT structure version?

Reply:
The mkey indexing I'd like to keep derivable from the structure itself, as new ABI versions are published. I don't want to assign any strange ordering based on what I deem more important.